src/anomalib/data/base/dataset.py

```diff
@@ -117,24 +118,21 @@ def __getitem__(self, index: int) -> dict[str, str | torch.Tensor]:
         mask_path = self._samples.iloc[index].mask_path
         label_index = self._samples.iloc[index].label_index
 
-        image = read_image(image_path)
+        image = to_tensor(Image.open(image_path))
```

Comment:
Would it be an idea to modify the `read_image` function to replace the cv2 stuff with Image.open? This would then be something like this

```python
image = read_image(image_path, to_tensor=True)
```

Reply:
Another question, now that we are reading the images with PIL, this would mean that we are reading based on RGB? Would this mean that we need to check the visualizer stuff that converts rgb-2-bgr?

Reply:
Maybe we also check some other files (notebooks, inference tools, etc.) where the `read_image` function is used.

![Screenshot_20240203_082656_Chrome](https://github.com/openvinotoolkit/anomalib/assets/5786636/2eae367e-cc87-4fc1-a95c-34ceb7d759d3)

Reply:
Same for `cv2.imread`

![Screenshot_20240203_084231_Chrome](https://github.com/openvinotoolkit/anomalib/assets/5786636/85866c51-8736-488b-9be1-39e269a33157)

Reply:
Can we use `read_image` here too?
```suggestion
        image = read_image(image_path)
```